lib/src/facts/windows/ssh_resolver.cc

```diff
@@ -0,0 +1,122 @@
+#include <internal/facts/windows/ssh_resolver.hpp>
+#include <leatherman/file_util/file.hpp>
+#include <facter/util/string.hpp>
+
+#include <leatherman/util/environment.hpp>
+#include <leatherman/windows/file_util.hpp>
+#include <leatherman/windows/system_error.hpp>
+#include <leatherman/windows/user.hpp>
+#include <leatherman/windows/windows.hpp>
+#include <leatherman/logging/logging.hpp>
+#include <boost/algorithm/string.hpp>
+#include <leatherman/logging/logging.hpp>
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+#include <boost/format.hpp>
+#include <tuple>
+#include <vector>
+
+#ifdef USE_OPENSSL
+#include <internal/util/scoped_bio.hpp>
+#include <openssl/sha.h>
+#include <openssl/evp.h>
+using namespace facter::util;
+#endif  // USE_OPENSSL
+
+using namespace std;
+using namespace facter::util;
+using namespace boost::filesystem;
+
+namespace bs = boost::system;
+namespace lth_file = leatherman::file_util;
+
+using namespace leatherman::util;
+
+namespace facter { namespace facts { namespace windows {
+
+    ssh_resolver::data ssh_resolver::collect_data(collection& facts)
```

Comment:
check if common code can be extracted in a file in parent dir instead duplicate it, similar to collection.cc class

Reply:
Done.